plugins/ci/skills/manage-symptoms/manage_symptoms.py

```
@@ -0,0 +1,170 @@
+        with urllib.request.urlopen(url, timeout=30) as resp:
+            return json.loads(resp.read().decode("utf-8"))
+    except (urllib.error.HTTPError, urllib.error.URLError) as e:
+        print("Error fetching %s: %s" % (url, e), file=sys.stderr)
+        return None
+
+
+def check_labels_exist(label_ids):
+    labels = get_json(READ_LABELS_URL)
+    if not isinstance(labels, list) or not all(isinstance(l, dict) for l in labels):
+        return ["could not verify label IDs (labels API unreachable or returned unexpected data)"]
+    known = {l.get("id") for l in labels}
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)